mrp_operations_start_without_material/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Daniel Campos
+# © 2015 Pedro M. Baeza
+# © 2015 Ana Juaristi
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP Operations start without material",
+    "version": "8.0.1.0.1",
+    "author": "OdooMRP team,Odoo Community Association (OCA)",
```

Comment:
Please include here Tecnativa, AvanzOSC, your company, and you can remove OdooMRP team if you want.